src/scratchtocatrobat/converter/converter.py

```diff
@@ -354,6 +355,15 @@ def create_script(cls, scratch_script_name, arguments, catrobat_project, sprite,
             my_script.formulaMap = when_cond_brick.formulaMap
             return my_script
 
+        if scratch_script_name == scratch.SCRIPT_WHEN_BACKGROUND_SWITCHES_TO:
+            background = catrobat.background_sprite_of(catrobat_project.getDefaultScene())
```

Comment:
Added the mentioned line and a test.
It seems a bit odd to me to squash every change into one commit.
I think we should make new commits for changes to a Pull-Request.